app/client-v2/packages/ui/components/MapByLayer.tsx

```diff
@@ -1,6 +1,8 @@
 import "mapbox-gl/dist/mapbox-gl.css";
 import React from "react";
-import Map, { Layer, Source } from "react-map-gl";
+import Map, { Layer,  Source } from "react-map-gl";
+import DeckGL from "@deck.gl/react/typed";
+import { _WMSLayer as WMSLayer } from "@deck.gl/geo-layers/typed";
```

Comment:
I think it would be better to use the WMSLayer from maplibre. We can use Deck.gl later for other visualization but for now we can just stay with React-Map-gl + Maplibre/Mapbox.